.github/workflows/build-docs.yml

```diff
@@ -6,12 +6,6 @@ on:
       - master
   pull_request:
 
-permissions:
-  deployments: write
-  issues: write
-  pull-requests: write
-  statuses: write
```

Comment:
@lparadkar-rocket 
Please revert these changes
Also, please avoid changing workflows without discussing with other squads 😋

Reply:
Agreed, but I don't see in the list of commits where these permissions were removed in the first place.

Reply:
@lparadkar-rocket, I'm sorry for incorrectly tagging you.

After looking through the history a bit more closely, I noticed that it was someone else.
My apologies 😢